src/js/i18n/en-US.yml

```
@@ -824,6 +824,9 @@ careOptsFrontend:
           headingView:
             headingText: Program Action
             headingOutreachText: Program Outreach Action
+          menuView:
+            delete: Delete Program Action
+            headingText: Program Action Menu  
```

yamllint

[error] trailing spaces

(trailing-spaces)